src/main/java/com/spotify/docker/client/messages/Network.java

```diff
@@ -117,4 +119,19 @@ static Container create(
       return new AutoValue_Network_Container(endpointId, macAddress, ipv4Address, ipv6Address);
     }
   }
+  
+  /**
+   * Docker networks come in two kinds: built-in or custom. 
+   */
+  public enum Type {
+    /** Predefined networks that are built-in into Docker. */
+    BUILTIN,
+    /** Custom networks created by users. */
+    CUSTOM;
+    
+    @Override
+    public String toString() {
+      return name().toLowerCase(Locale.ENGLISH);
```

Comment:
@johnflavin `Network.Type` is not used in JSON. It is just syntactical sugar so that you don't have to worry about the possible values for `type`. The enum is converted to a string almost immediately. But usage might change at some point, so it's a good idea to make it JSON friendly. I'll adjust it to `Event.Type`.